intf/vernacexpr.mli

```diff
@@ -283,14 +283,9 @@ type bullet =
     | Plus of int
 
 (** {6 Types concerning Stm} *)
-type 'a stm_vernac =
+type stm_vernac =
   | JoinDocument
-  | Finish
   | Wait
```

Comment:
Enrico my impression was that `fake_ide` only used `WAIT` and `JOIN`, but I'll double-check.

Note that the test-suite passes here, whereas the debacle in #360 is that we merged it without even checking the test-suite.